lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -532,14 +533,17 @@ def test_get_threads_by_topic_id(self):
 
     def test_basic_query_params(self):
         self.get_thread_list([], page=6, page_size=14)
-        self.assert_last_query_params({
-            "course_id": [unicode(self.course.id)],
-            "sort_key": ["date"],
-            "sort_order": ["desc"],
-            "page": ["6"],
-            "per_page": ["14"],
-            "recursive": ["False"],
-        })
+        self.assert_last_query_params(
```

Comment:
Style nit: The convention in the code seems to be putting the opening parenthese and curly brace on the same line.